Modix.Services/Notes/NoteRemoverService.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Modix.Services.Notes
+{
+    using System;
+    using System.Threading.Tasks;
+    using Data;
+    using Data.Models.Moderation;
+    using Serilog;
+
+    public class NoteRemoverService : INoteRemoverService
+    {
+        private readonly ModixContext _modixContext;
+
+        public NoteRemoverService(ModixContext modixContext)
+        {
+            _modixContext = modixContext;
+        }
+
+        public async Task<bool> Remove(NoteEntity note)
+        {
+            try
+            {
+                _modixContext.Notes.Remove(note);
+                var rowsChanged = await _modixContext.SaveChangesAsync();
+
+                return rowsChanged > 0;
```

Comment:
Again, not big on returning a `bool` to indicate success or failure instead of letting the exception bubble up.

Reply:
Also, I don't think it's necessarily an error if a note record wasn't deleted because it never existed in the first place.